tsdb/wlog/checkpoint.go

```diff
@@ -81,8 +83,8 @@ func DeleteCheckpoints(dir string, maxIndex int) error {
 	return errs.Err()
 }
 
-// CheckpointPrefix is the prefix used for checkpoint files.
-const CheckpointPrefix = "checkpoint."
```

Comment:
This constant was explicitly exported in https://github.com/prometheus/prometheus/pull/15636 , I don't think we should change the visibility. Also it makes the diff bigger.